src/sentry/api/endpoints/organization_member_details.py

```diff
@@ -204,6 +199,12 @@ def put(self, request, organization, member_id):
             if not result['role'] in {r.id for r in allowed_roles}:
                 return Response(
                     {'role': 'You do not have permission to invite that role.'}, status=403)
+                # You can't edit your own role
```

Comment:
nit, it'd move this to be right above the block it's describing and fix the indentation